app/models/course_session.rb

```diff
@@ -18,7 +18,7 @@ class CourseSession < ApplicationRecord
 
   class << self
     def accepting_applications
-      CourseSession.where("status = ?",statuses["admissions"]).
+      CourseSession.where("status = ?", statuses["admissions"]).
```

Comment:
where `statuses` coming from? is it `enum` that defines it ?

Reply:
Yeah, it's from the enum, I found this curious as well.